zcash_primitives/src/transaction/builder.rs

```diff
@@ -219,13 +219,13 @@ impl TransparentInputs {
         consensus_branch_id: consensus::BranchId,
     ) {
         let mut sighash = [0u8; 32];
-        for (i, info) in self.inputs.iter().enumerate() {
+        for (index, info) in self.inputs.iter().enumerate() {
             sighash.copy_from_slice(&signature_hash_data(
                 mtx,
                 consensus_branch_id,
                 SIGHASH_ALL,
-                Some((i, &info.coin.script_pubkey, info.coin.value)),
-                // tze equivalent is ???
+
+                SignableInput::Transparent{ index, script_code: &info.coin.script_pubkey, value: info.coin.value},
```

Comment:
```suggestion
                SignableInput::transparent(index, &info.coin.script_pubkey, info.coin.value),
```

Reply:
With this, you can revert the `i -> index` rename as well, slightly reducing the diff.